src/app/api/supabaseClient/__mocks__/index.ts

```
@@ -0,0 +1,84 @@
+    }
+  },
+  insert: (
+    tableId: keyof typeof mockDB.tables,
+    data: Record<string, string | number>,
+    error: Error | null = null
+  ) => {
+    const newRow = {
+      id: mockDB.tables[tableId].length,
+      ...Object.entries(data).reduce(
+        (acc, [key, value]) => ({
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)